mesonbuild/build.py

```diff
@@ -1770,6 +1770,122 @@ def get(self, lib_type: T.Literal['static', 'shared']) -> LibTypes:
         """Base case used by BothLibraries"""
         return self
 
+    def determine_rpath_dirs(self) -> T.Tuple[str, ...]:
```

Comment:
If you are going to make this refactor, you need to do it *before* adding code using the old location :P don't just refactor and remove that new code.

Reply:
Commit message also doesn't include a "why".